python_modules/dagster/dagster/core/definitions/inference.py

```diff
@@ -52,6 +59,81 @@ def infer_input_definitions_for_lambda_solid(solid_name, fn):
     return _infer_inputs_from_params(params, '@lambda_solid', solid_name)
 
 
+def _infer_output_definitions_from_docstring(solid_name, fn):
+    if not is_module_available("docstring_parser"):
+        return []
+
+    from docstring_parser import parse
+
+    docstring = parse(fn.__doc__)
+
+    if docstring.returns is None:
+        return []
```

Comment:
we should return explicit `None` here since `[]` is a valid value for `output_defs`